packages/codegen-ui-golden-files/lib/react/forms/datastore-create-form-with-many-to-many-relationship/TagCreateForm.jsx

```diff
@@ -0,0 +1,397 @@
+/** *************************************************************************
+ * The contents of this file were generated with Amplify Studio.           *
+ * Please refrain from making any modifications to this file.              *
+ * Any changes to this file will be overwritten when running amplify pull. *
+ ************************************************************************* */
+
+/* eslint-disable */
+import * as React from 'react';
+import { fetchByPath, validateField } from './utils';
+import { Tag, Post, TagPost } from '../models';
+import { getOverrideProps, useDataStoreBinding } from '@aws-amplify/ui-react/internal';
+import {
```

Comment:
can we hold off on merging til main gets merged in?

Reply:
Sure